src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -64,6 +66,7 @@ class_exists(MissingDotenv::class, false) || class_exists(Dotenv::class) || clas
  *  - int|string|null as handled by GenericRuntime.
  *
  * @author Nicolas Grekas <p@tchwork.com>
+ * @author Kévin Dunglas <kevin@dunglas.dev>
```

Comment:
nothing personal here, but there has to be some rules to allow adding lines on this file
the one I have in mind is: as a contributor to Symfony, who should I ping to get support on this class
I'm not sure Kevin would be happy to get asked about the logic in this class :) 
and the added lines don't need his specific support either
:thinking: